puppet/modules/redmine/manifests/init.pp

```diff
@@ -169,17 +169,11 @@
       passenger_start_timeout => 600,
     }
   } else {
-    systemd::manage_unit {'redmine.socket':
+    systemd::unit_file {'redmine.socket':
```

Comment:
`manage_unit` was not enough?

Reply:
in 6.0.0 it is enough. but the amount of modules I'd need to fix/bump is too high for now.

Reply:
Ah, fair enough.

Reply:
that's what the `make code compatible with systemd 4.2.0` commit is for -- it can be simply reverted once we have 6.0

Reply:
the revert is tracked below as an todo item